src/github.com/couchbase/sync_gateway/base/bucket.go

```diff
@@ -343,6 +343,10 @@ func GetCouchbaseBucket(spec BucketSpec) (bucket Bucket, err error) {
 		// Start bucket updater - see SG issue 1011
 		cbbucket.RunBucketUpdater(func(bucket string, err error) {
 			Warn("Bucket Updater for bucket %s returned error: %v", bucket, err)
+
+			if(callback != nil) {
+				callback(bucket, err)
+			}
```

Comment:
Does BucketUpdater failure actually cover the 'lost TAP feed' scenarios?  In the original issue related to #722, the TAP feed was dropped, but other operations against the bucket (e.g. writes) continued to succeed.